server/protocols/webadmin/src/test/java/org/apache/james/webadmin/WebAdminConfigurationTest.java

```diff
@@ -46,7 +46,7 @@ public void buildShouldWorkWithoutPortWhenDisabled() {
         assertThat(WebAdminConfiguration.builder()
             .disabled()
             .build())
-            .isEqualTo(new WebAdminConfiguration(false, null, HttpsConfiguration.builder().disabled().build()));
+            .isEqualTo(new WebAdminConfiguration(false, null, HttpsConfiguration.builder().disabled().build(), false, "*"));
```

Comment:
assertions become unreadable because there's too many parameters, it's time to extract the field you actually want to assert